core/include/traccc/edm/cell.hpp

```diff
@@ -85,4 +85,17 @@ using cell_collection_types = collection_types<cell>;
 /// Declare all cell container types
 using cell_container_types = container_types<cell_module, cell>;
 
+/// Alternative cell structure which contains both the cell and its module.
+///
+/// This can be used for storing all information in a single collection, whose
+/// objects need to have both the header and item information from the container
+/// types defined above.
+struct alt_cell {
+    cell c;
+    cell_module module;
```

Comment:
Actually this was quite a useful comment. I did not use a link because in our current implementation the cell_module struct contains:
```c++
channel_id range0[2] = {std::numeric_limits<channel_id>::max(), 0};
channel_id range1[2] = {std::numeric_limits<channel_id>::max(), 0};
```
which is defined as the limits of the cells' positions belonging to this module, so you'd need to hold the module in memory and update these values as you read new cells. 
However, it turns out that we're not using these anywhere, so we might as well just entirely remove this from our EDM, no?

Reply:
Seems rather useless to me. If you want the actual size (i.e. number of pixels) of the module you can get those as static properties of the modules themselves, the range in which activations lie seems&mdash;to me&mdash;rather useless.